pyiron_contrib/workflow/io.py

```diff
@@ -206,6 +215,12 @@ class InputSignals(SignalIO):
     def _channel_class(self) -> type(InputSignal):
         return InputSignal
 
+    def disconnect_run(self) -> list[tuple[Channel, Channel]]:
+        try:
+            return self.run.disconnect_all()
```

Comment:
Where does this `run` come from?

Reply:
`__getattr__` on this class just looks in the dictionary of owned channels, so this is disconnecting the channel named `run` if it exists. This is _useful_ because every `Node` adds a `signals.input.run` and `signals.output.ran` to itself at instantiation.

In the long run it might be nice to do something cute [like Marvin did here](https://github.com/liamhuber/atomistics/blob/1a6dbfe96600b82cf7db4e4438cc995065a543d8/atomistics/calculators/interface.py#L15) and have the "immortal channels" rigorously defined somewhere. In the meantime, while I admit that the usefulness is opaque without knowledge of other classes, I'm comfortable with this because the behaviour is so explicit -- `disconnect_run` attempts to disconnect run 😂